wire-ios/Wire-iOS/Sources/Components/Settings/SettingsPropertyFactory.swift

```diff
@@ -422,6 +424,22 @@ final class SettingsPropertyFactory {
                 }
             )
 
+        case .collapseOwnMessages:
+            guard let userId = selfUser?.remoteIdentifier else {
+                fatalError()
+            }
```

Comment:
suggestion: `fatalError` without a message is less useful than a force-unwrapping error.
```suggestion
            let userId = selfUser!.remoteIdentifier!
```